mantidproject/mantid
# Remove nist save option

**Description of work.**
The Nist data format is no longer needed, this removes the option to use it from the GUI's but leaves the algorithm for now.
**Report to:** stephen.king@stfc.ac.uk 

**To test:**

The things to test here are that the save NIST option does not appear on either GUI but also more importantly that the other save options still function properly. An example work-flow on how to do this is below. 

Get this data
[loqdemo.zip](https://github.com/mantidproject/mantid/files/2118722/loqdemo.zip)
For both GUI's
* Open the GUI, select LOQ as the instrument
* Load MaskFile.txt as the user file 
* Load batch_mode_reduction.csv as the batch file

In old GUI
* Click 1D reduce
* Select save other and save out the reduced workspace (line_one_reduction). Check that it has saved out without error. The files should be written to the output directory set in save file dialog. 

In new GUI
* Below the data table select Both as the output format.
* Click process, confirm that the reduced files are saved out.




Fixes #22631 . 

Does this update require release notes?
Requires release notes are here #22628 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
